lib/ui/views/new_dish/new_dish_view.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:receipe_app/generated/l10n.dart';
+import 'package:receipe_app/ui/common/app_images.dart';
+import 'package:receipe_app/ui/common/ui_helpers.dart';
+import 'package:receipe_app/ui/utilities/validation.dart';
+import 'package:receipe_app/ui/views/new_dish/new_dish_view.form.dart';
+import 'package:receipe_app/ui/widgets/common/primary_button/primary_button.dart';
+import 'package:stacked/stacked.dart';
+import 'package:receipe_app/ui/extension/app_typography.dart';
+import 'package:receipe_app/ui/extension/palette.dart';
+import 'package:stacked/stacked_annotations.dart';
+import 'new_dish_viewmodel.dart';
+
+@FormView(fields: [
+  FormTextField(name: 'dish'),
+  FormTextField(name: 'instruction'),
+  FormTextField(name: 'ingredient')
+])
+class NewDishView extends StackedView<NewDishViewModel>with $NewDishView {
+  const NewDishView({Key? key}) : super(key: key);
+
+  static final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+
+
+  @override
+  Widget builder(
+    BuildContext context,
+    NewDishViewModel viewModel,
+    Widget? child,
+  ) {
+    final ThemeData theme = Theme.of(context);
+    final AppTypography? typography = theme.extension<AppTypography>();
+    final Palette? palette = theme.extension<Palette>();
+
+    return Scaffold(
+      appBar: AppBar(
+      leading:Align(
+                alignment: Alignment.center,
+                child: Padding(
+                  padding: EdgeInsets.only(
+                    left: 16.w,
+                    top: 8.h,
+                  ),
+                  child: SvgPicture.asset(
+                    AppImages.burgerLog,
+                    width: 24.7.w,
+                    height: 24.h,
+                  ),
+                ),
+              ),
+      title:Text(S.current.create_a_dish,
+                style: typography?.titleBold16?.copyWith(color: palette?.gray11)) ,     
+      ),
+      body: SingleChildScrollView(
```

Comment:
Let's try as much as possible to reduce the amount of widgets we are calling. For your scrolling, you can make use of a `ListView` which gives you `padding` automatically than what you have done here.

Reply:
Tricky thing here is that while list view is good, single child and column is better when using forms. List view "deletes" widget not in view, so imagine there's a validation error somewhere up the widget tree, and you scroll bk up to find it, you won't see it